src/workerd/api/node/node.h

```diff
@@ -125,6 +125,10 @@ void registerNodeJsCompatModules(Registry& registry, auto featureFlags) {
       return featureFlags.getEnableNodeJsVmModule();
     }
 
+    if (module.getName() == "node:v8"_kj) {
```

Comment:
As this list is growing, would it make sense to switch to:

```c++
auto moduleName = module.getName();

switch (moduleName) {
  case "node:vm"_kj:
    return featureFlags.getEnableNodeJsV8Module();

  // ... add cases for 1:1 name - flag

  default:
    // handle "complex" cases where multiple names map to a single flag
    if (isNodeHttpModule(module.getName())) {
      return featureFlags.getEnableNodejsHttpModules();
    }
    // ...

    // fallback to true
}

Reply:
I'll be coming back in and refactoring this list a bit later. For now I'll leave it as is.